MIGRATION.md

```diff
@@ -337,9 +335,11 @@ class MyEntityRepositorySpec extends DefaultPlayMongoRepositorySupport[MyEntity]
 }
 ```
 
-`DefaultPlayMongoRepositorySupport` will ensure that the database is cleaned and setup (with indexes and schemas) before each test, and turn on `no table scan` to ensure all queries have an index defined. To refine this behaviour, you may use the composed traits directly.
+`DefaultPlayMongoRepositorySupport[T]` requires you to override `repository` in your tests, and provides a `mongoComponent` to create the repository. This will create a database named after the test.
+
+**NB** - Unlike the old `MongoSpecSupport`, `DefaultPlayMongoRepositorySupport` will ensure that the database is **cleaned of all data** and setup (with indexes and schemas) before each test, and turn on `no table scan` to ensure all queries have an index defined. To refine this behaviour, you may use the composed traits directly.
```

Comment:
We could be clearer about the extra behaviour coming from additional traits?
e.g.

"**NB** In addition to initialising a mongoComponent (as per `MongoSpecSupport` from reactivemongo-test), `DefaultPlayMonoRepositorySupport` also includes additional traits (akin to `RepositoryPreparation` and `FailOnUnindexedQueries` from reactivemongo-test) which ensure ..."